auth-service/src/main/java/com/auth/authservice/businesslayer/UserServiceImpl.java

```diff
@@ -202,9 +205,197 @@ public HashMap<String,Object> login(UserIDLessUsernameLessDTO login) throws Inco
+            throw new InvalidInputException("Invalid token");
+        } else {
+            updatePassword(password, token);
+        }
+    }
+
+
+    public void sendEmailForgotPassword(String recipientEmail, String link){
+
+
+        // Initialize an empty HTMLDocument object instance
+             String email = format("""
+                     <!DOCTYPE html>
+                     <html lang="en">
+                     <head>
+                         <meta charset="UTF-8">
+                         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+                         <title>Password Reset</title>
+                         <style>
+                             body {
+                                 font-family: Arial, sans-serif;
+                                 background-color: #f4f4f4;
+                                 margin: 0;
+                                 padding: 0;
+                             }
+                             .container {
+                                 max-width: 600px;
+                                 margin: 0 auto;
+                                 padding: 20px;
+                                 background-color: #fff;
+                                 border-radius: 5px;
+                                 box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+                             }
+                             h1 {
+                                 color: #333;
+                             }
+                             p {
+                                 color: #555;
+                             }
+                             a {
+                                 color: #007BFF;
+                             }
+                         </style>
+                     </head>
+                     <body>
+                         <div class="container">
+                             <h1>Reset Your Password - Verification Code</h1>
+                             <h3>We have received a request to reset your password for your Pet Clinic account. To ensure the security of your account, please follow the instructions below to reset your password.</h3>
+                            \s
+                             <ol>
+                                 <li>Click on the following link to access the password reset page: <a href="%s">Reset Password</a></li>
+                                 <li>Follow the on-screen instructions to create a new password for your account.</li>
+                             </ol>
+                            \s
+                             <p>If you did not request this password reset, please disregard this email. Your account security is important to us, and no changes will be made without your verification.</p>
+                            \s
+                             <p>Thank you for choosing Pet Clinic.</p>
+                         </div>
+                     </body>
+                     </html>
+                     """, link);
+
```

Comment:
is the email responsive im just thinking for the people who open this on their phone can they read it and make it work?

Reply:
this is a detail i think if its not you should add it to your sprint backlog

Reply:
It is ! There is little to no styling, but I am thinking of making it better.

Reply:
thats good excited to hear more